classes/class-wpcom-liveblog-entry-extend-feature-hashtags.php

```diff
@@ -39,6 +39,9 @@ public function load() {
 		// the generated hashtag class.
 		$this->class_prefix = apply_filters( 'liveblog_hashtag_class', $this->class_prefix );
 
+		// Set a better regex for hashtags to allow for hex values in content.
+		$this->set_regex( '~(?:(?<!\S)|>?)(?:(?<!&|&amp;))((?:' . implode( '|', $this->get_prefixes() ) . ')([0-9_\-\p{L}]*))~um' );
```

Comment:
Where has this regex come from? Where's the unit tests to show it working? Where's the documentation / breakdown, like [here](https://github.com/WordPress/WordPress/blob/da7a80d67fea29c2badfc538bfc01c8a585f0cbe/wp-includes/shortcodes.php#L243).

I've no doubt that it works, but for the sake of reducing technical debt moving forward, I'd like to see this PR be rounded out more.